appserver/tests/appserv-tests/devtests/deployment/jsr88/misc/exampleFromJ2EETutorial/cb/server/web/errorpage.txt

```diff
@@ -1,31 +1,45 @@
-<%--
- * Copyright (c) 2003 Sun Microsystems, Inc.  All rights reserved.  U.S. 
- * Government Rights - Commercial software.  Government users are subject 
- * to the Sun Microsystems, Inc. standard license agreement and 
- * applicable provisions of the FAR and its supplements.  Use is subject 
- * to license terms.  
- * 
- * This distribution may include materials developed by third parties. 
- * Sun, Sun Microsystems, the Sun logo, Java and J2EE are trademarks 
- * or registered trademarks of Sun Microsystems, Inc. in the U.S. and 
- * other countries.  
- * 
- * Copyright (c) 2003 Sun Microsystems, Inc. Tous droits reserves.
- * 
- * Droits du gouvernement americain, utilisateurs gouvernementaux - logiciel
- * commercial. Les utilisateurs gouvernementaux sont soumis au contrat de 
- * licence standard de Sun Microsystems, Inc., ainsi qu'aux dispositions 
- * en vigueur de la FAR (Federal Acquisition Regulations) et des 
- * supplements a celles-ci.  Distribue par des licences qui en 
- * restreignent l'utilisation.
- * 
- * Cette distribution peut comprendre des composants developpes par des 
- * tierces parties. Sun, Sun Microsystems, le logo Sun, Java et J2EE 
- * sont des marques de fabrique ou des marques deposees de Sun 
- * Microsystems, Inc. aux Etats-Unis et dans d'autres pays.
+<%@ page isErrorPage="true" %>
+ NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
```

Comment:
Ok, I'm going to assume you'll find all the instances of this problem and fix them all.